### PR TITLE
Add code styling tools

### LINKED ISSUE #23: Code cleanup
Hello @nacimgoura, thanks for creating this tool!

How do you feel about adding prettier + editor config to the project? If that sounds good to you, I can create a PR.

Thanks!

### PR DESCRIPTION
**Pull request by alejandronanez**
Hey @nacimgoura.

I added a couple of things here:
- Editorconfig
- Eslint
- Prettier
- A pre-commit hook that runs prettier to all staged files
- Run prettier on all files

After this, everything should be consistent in terms of code styling.

**NOTE**: This is using 2 spaces which IMO looks more readable than what we had before, but if you don't like it I can revert this change.

closes #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,27 +9,31 @@
...
+
 Automation Script for crawling information from multiple instagram profile,
 like the number of posts, followers, tags and mentions of the posts.
 Inspired by [this project](https://github.com/timgrossmann/instagram-profilecrawl) but made in nodeJS and more complete.
 
 Two way to crawl profile data in Instagram :
-- use Instagram API (the faster)
-- use Selenium with chromedriver (more complete but slower)
+
+* use Instagram API (the faster)
```
Comment (alejandronanez) [inline README.md]: Prettier also formats `*.md` files :)

File: crawl.js [hunk 2]
```diff
@@ -21,250 +25,275 @@ let listProfileName = [];
  * Init selenium with chrome on headless mode
  */
 module.exports = {
-	start(listName) {
-		spinnerCrawl.start();
-		listProfileName = listName;
-		selenium.start(err => {
-			if (err) {
-				console.log(err);
```
Comment (alejandronanez) [inline crawl.js]: I removed this `console.log` since it was throwing an eslint error.

**Comment by nacimgoura** [comment]
Thanks but I already use [xo](https://github.com/xojs/xo) for the lint. Why used eslint instead of xo would be better?

**Comment by alejandronanez** [comment]
Oops, I missed XO on your devDependencies.

Let’s stick with it, XO uses eslint under the hood. I’ll update the PR then.

**Comment by nacimgoura** [comment]
Thanks 😉

**Pull request merged**